frontend/packages/dev-console/src/components/import/pac/PacSection.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import { ValidatedOptions, Alert } from '@patternfly/react-core';
+import { FormikValues, useFormikContext } from 'formik';
+import { useTranslation } from 'react-i18next';
+import FormSection from '@console/dev-console/src/components/import/section/FormSection';
+import { getGitService, GitProvider, ImportStrategy } from '@console/git-service/src';
+import { Loading } from '@console/internal/components/utils';
+import {
+  usePacInfo,
+  useRepositoryPresent,
+} from '@console/pipelines-plugin/src/components/repository/hooks/pac-hook';
+import { recommendRepositoryName } from '@console/pipelines-plugin/src/components/repository/repository-form-utils';
+import ConfigTypeSection from '@console/pipelines-plugin/src/components/repository/sections/ConfigTypeSection';
+import WebhookSection from '@console/pipelines-plugin/src/components/repository/sections/WebhookSection';
+import './PacSection.scss';
+
+const PacSection: React.FC = () => {
+  const { t } = useTranslation();
+  const formContextField = 'pac.repository';
+  const [githubAppAvailable, setGithubAppAvailable] = React.useState(false);
+  const { values, setFieldValue, errors, setFieldError } = useFormikContext<FormikValues>();
+  const {
+    import: { strategies, recommendedStrategy },
+    git: { url, type, ref, dir, secretResource },
+  } = values;
+  const [validated, setValidated] = React.useState<ValidatedOptions>(ValidatedOptions.default);
+  const [pac, loaded] = usePacInfo();
+  const repoAlreadyExists = useRepositoryPresent(url);
+
+  const handlePacChange = React.useCallback(async () => {
+    const gitService = getGitService(url, type, ref, dir, secretResource);
+    const isTektonFolderPresent = gitService && (await gitService.isTektonFolderPresent());
+    if (isTektonFolderPresent) {
```

Comment:
@Lucifergene Both `.tekton` folder and in it atleast one PLR yaml file should be present.

Reply:
Have added the mentioned condition